### DISCUSSION TITLE
Updating of $LL elements in arrays without overhead / reasigning

### DISCUSSION THREAD

**Discussion by MDoder97** [ivanhofer/typesafe-i18n#412] [Q&A] [answered]
Hey, I'm using sveltekit

So if I have a list of items
`let items = [$LL.itemOne(), $LL.itemTwo(), $LL,itemThree()]`
I notice that the items aren't automcatically updated without me doing something like
`$: $locale, items = items`

Is there way to do this without the extra overhead? The app I'm working on has large amounts of menu items that are stored in a separate index.ts files in order to not pollute component clarity too much.

I understand this is a sveltekit and js issue and not typesafe, still wondering if something can be done internally for this to work.

Great job on the library, thanks.

**Comment by ivanhofer** [comment, marked as answer]
It works like expected. You are not defining the array in a reactive way. Just replace `let items` with `$: items` and the array will uodate whenever something inside `$LL` changes.
